IdentityCore/src/broker_operation/response/MSIDBrokerNativeAppOperationResponse.m

```diff
@@ -67,7 +67,7 @@ + (NSString *)responseType
 
 - (NSNumber *)httpStatusCode
 {
-    if (!_httpStatusCode) _httpStatusCode = @200;
+    if (_httpStatusCode == nil) _httpStatusCode = @200;
```

Comment:
nit: do we really want to do `== nil`? Any way to suppress this warning?

Reply:
We don't need to follow this pattern for other cases, seems anylizer throws this error only for NSNumbers:

`Converting a pointer value of type 'NSNumber * _Nonnull' to a primitive boolean value; instead, either compare the pointer to nil or call -boolValue`